src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/visit/FuturePublicVisitsTest.kt

```diff
@@ -38,48 +33,32 @@ class PublicVisitsTest : IntegrationTestBase() {
 
   private lateinit var otherSessionTemplate: SessionTemplate
 
-  private lateinit var visit: Visit
+  private lateinit var nearestVisit: Visit
+  private lateinit var visitFarInTheFuture: Visit
   private lateinit var visitInDifferentPrison: Visit
   private lateinit var visitWithOtherBooker: Visit
 
   @BeforeEach
   internal fun createVisits() {
     otherSessionTemplate = sessionTemplateEntityHelper.create(prisonCode = "AWE")
 
-    visitInDifferentPrison = createApplicationAndVisit(prisonerId = "AA0000A", slotDate = startDate.plusWeeks(1), sessionTemplate = otherSessionTemplate, userType = PUBLIC)
-    visitInDifferentPrison = visitEntityHelper.save(visitInDifferentPrison)
+    visitFarInTheFuture = createVisit(prisonId = "visit far away", actionedByValue = "aTestRef", VisitStatus.BOOKED, sessionTemplateDefault, PUBLIC, 6)
```

Comment:
suggestion: Could we use named parameters? Makes it easier to see whats being set when reviewing tests at a glance.